deploy/example_catalog/cr-app-flask-inf-engine.json

```diff
@@ -0,0 +1,121 @@
+{
```

Comment:
also, rename cr-app-flask-inf-engine -> cr-app-deployment-engine to avoid confusion